packages/@react-aria/gridlist/src/useGridListItem.ts

```diff
@@ -106,11 +106,24 @@ export function useGridListItem<T>(props: AriaGridListItemOptions, state: ListSt
       if (parent) {
         // siblings must exist because our original node exists, same with lastItem
         let siblings = state.collection.getChildren?.(parent.key)!;
-        setSize = getLastItem(siblings)!.index + 1;
+        let lastItemOfParent = getLastItem(siblings)!;
+        // Don't include loaders as part of the item count, to be revisited when we support focusing the loaders
+        // Also assumes only one loader per tree parent row
```

Comment:
and assumes the loader is the last child. doesn't the tree item content node also count as a child of tree items? would it be safer to just count all the item children?

Reply:
yeah, thats true. Counting just the items should work, will update